compositor_common/src/validators.rs

```diff
@@ -249,6 +252,18 @@ impl SceneSpec {
 
         Ok(())
     }
+
+    fn validate_transforms_params(&self) -> Result<(), SpecValidationError> {
+        for TransformNodeSpec {
+            node_id,
+            transform_params,
+            ..
+        } in self.transforms.iter()
+        {
```

Comment:
```suggestion
        for t in self.transforms.iter() {
            TransformNodeSpec {
                node_id,
                transform_params,
                ..
            } = t;
```
?

Reply:
With `FittedColumn`, the resolution is modeled on the type system, so I removed this check entirely